build.sbt

```
@@ -1,21 +1,22 @@
 import uk.gov.hmrc.DefaultBuildSettings
 
 ThisBuild / majorVersion := 0
-ThisBuild / scalaVersion := "2.13.12"
+ThisBuild / scalaVersion := "3.3.6"
 
 lazy val microservice = Project("crdl-ref-data-dps-stub", file("."))
   .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
+  .settings(CodeCoverageSettings.settings: _*)
```

I think this should probably be converted to the Scala 3 syntax or we will get warnings about `-Xsource:3`
```suggestion
  .settings(CodeCoverageSettings.settings *)
```